templates/graphql-storyblok/grafbase/grafbase.config.ts

```diff
@@ -0,0 +1,31 @@
+import { g, connector, config } from '@grafbase/sdk'
+
+const storyblok = connector.GraphQL('Storyblok', {
+  url: 'https://gapi.storyblok.com/v1/api',
+  headers: (headers) => {
+    headers.set('Token', { forward: 'Token' })
+    headers.set('Version', { forward: 'Version' })
+  }
+})
+
+g.datasource(storyblok)
+
+// Disabling namespace may cause conficts with other connectors
+// g.datasource(storyblok, { namespace: false })
```

Comment:
remove

Reply:
This is inline with all other templates that show how to disable namespacing (all guides mention this too).

Reply:
Right, but the namespace argument goes in the GraphQL() section

Reply:
With SDL it does yeah